client/state/purchases/actions.js

```diff
@@ -0,0 +1,136 @@
+// External dependencies
+import i18n from 'i18n-calypso';
+
+// Internal dependencies
+import olark from 'lib/olark';
+import {
+	PRIVACY_PROTECTION_CANCEL,
+	PRIVACY_PROTECTION_CANCEL_COMPLETED,
+	PRIVACY_PROTECTION_CANCEL_FAILED,
+	PURCHASES_REMOVE,
+	PURCHASES_SITE_FETCH,
+	PURCHASES_SITE_FETCH_COMPLETED,
+	PURCHASES_SITE_FETCH_FAILED,
+	PURCHASES_USER_FETCH,
+	PURCHASES_USER_FETCH_COMPLETED,
+	PURCHASES_USER_FETCH_FAILED,
+	PURCHASE_REMOVE,
+	PURCHASE_REMOVE_COMPLETED,
+	PURCHASE_REMOVE_FAILED
+} from 'state/action-types';
+import purchasesAssembler from 'lib/purchases/assembler';
```

Comment:
At some point we should check if it would be possible to move `purchasesAssembler` call to selector to have only plain JS data in the state ;)